modules/test/helpers/common.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 Red Hat
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"time"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// TestHelper -
+type TestHelper struct {
+	k8sClient client.Client
+	ctx       context.Context
+	cancel    context.CancelFunc
```

Comment:
I'm not sure we need to hold the cancel here. It is not used by the helpers. It is only used by the test suite.

Reply:
yes, I just also added it as it is connected to the ctx as tuple. but yes when we pass the ctx as you stated bellow as a param we can remove it.

Reply:
done